wandb/apis/public.py

```diff
@@ -2926,6 +2926,65 @@ def _get_ref_artifact_from_entry(self, entry):
         artifact_id = util.host_from_path(entry.ref)
         return Artifact.from_id(util.hex_to_b64_id(artifact_id), self.client)
 
+    def used_by(self):
```

Comment:
The public API should be object-oriented. So both of these new methods should return a Runs object, not a list of run IDs.

Reply:
I updated these to be OO in the latest commit